ui/src/api/tasks/hooks.ts

```diff
@@ -23,3 +24,10 @@ export const useGetTaskQuery = (id: number) => {
     queryFn: () => getTask(id),
   });
 };
+
+export const useClassifyImageMutation = () => {
+  return useMutation<ClassifyImageResponse, Error, ClassifyImageRequest>({
```

Comment:
generalnie Error jest chyba trochę zbyt ogólny tutaj, ale chyba możemy to zostawić

Reply:
Error to domyślny typ, a backend za bardzo innych errorów jak 500 nie daje...